pyatlan/model/core.py

```diff
@@ -87,21 +84,26 @@ def __eq__(self, other):
         )
 
     @classmethod
-    def _convert_to_display_text(cls, data):
-        from pyatlan.cache.atlan_tag_cache import AtlanTagCache
-
+    def _convert_to_display_text(
+        cls,
+        data,
```

Comment:
Need to add client parameter otherwise you need to remember to set the correct default_client

Reply:
Can add, but unclear to me how this would ultimately be passed through via `__get_validators__` (which if it's always sending as the default `None` then would behave identically to not passing it through at all, I think?)

Reply:
And indeed, Pydantic throws an error if we pass through a client parameter:

```
pydantic.errors.ConfigError: Invalid signature for validator <bound method AtlanTagName._convert_to_display_text of <class 'pyatlan.model.core.AtlanTagName'>>: (data, client: 'Optional[AtlanClient]' = None), should be: (value, values, config, field), "values", "config" and "field" are all optional.
```